### DISCUSSION TITLE
Is there any guide or links available to self build an image as Release 23.01

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#288] [Questions] [answered]
@svpcom Hi, i have a quesition here.

Is there any guide or links available to self build an image as Release 23.01?

Any steps or scripts available for me?

**Comment by svpcom** [comment, marked as answer]
Yes, https://github.com/svpcom/wfb-ng/wiki/Setup-HOWTO

**Reply by svpcom** [reply]
All systemd scripts are in `scripts` directory

**Reply by lida2003** [reply]
> Yes, https://github.com/svpcom/wfb-ng/wiki/Setup-HOWTO

well, i mean how to build image.

**Reply by svpcom** [reply]
Use standard RPI image, then do all steps in howto, then capture image from sd-card

**Reply by lida2003** [reply]
> Use standard RPI image, then do all steps in howto, then capture image from sd-card

ok.
